meows-dctest-runner/neco-apps-bootstrap

```diff
@@ -1,6 +1,11 @@
 #!/bin/sh -xe
 
-git clone -b ${NECO_APPS_BRANCH:-release} https://github.com/cybozu-go/neco-apps.git ${NECO_APPS_DIR}
+set +x
+if [ "${GITHUB_USER}" != "" -a "${GITHUB_TOKEN}" != "" ]; then
```

Comment:
The `test` command has `-n` option to check the empty string. How about using `-n` option?

```suggestion
if [ -n "${GITHUB_USER}" -a -n "${GITHUB_TOKEN}" ]; then
```

```
$ man test
...
       -n STRING
              the length of STRING is nonzero
```

Reply:
Thank you.
Your comment has been reflected.